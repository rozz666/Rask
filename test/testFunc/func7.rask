f(int32 arg1, int32 arg2) -> void
{
    print(arg1);
    print(arg2);
}

main() -> void
{
    f(1, 2);
    var x = 10;
    f(x, 2);
    f(1, x);
    f(x, x);
}
