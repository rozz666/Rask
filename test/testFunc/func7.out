1
2
10
2
1
10
10
10

